pkgs/development/python-modules/releases/default.nix

```diff
@@ -0,0 +1,34 @@
+{ buildPythonPackage
+, lib
+, fetchPypi
+, sphinx
+, semantic-version
+}:
+
+buildPythonPackage rec {
+  version = "1.6.1";
+  pname = "releases";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0m6skhfdjmfgwn7f3dhzdwa6sxamy0fzbn319vf42b86mdik26vs";
+  };
+
+  propagatedBuildInputs = [ semantic-version sphinx ];
+
+  # Use compatible sphinx versions
+  prePatch = ''
+    substituteInPlace setup.py \
+      --replace "sphinx>=1.3,<1.8" "sphinx ~= 1.3"
+  '';
+
+  # Tests excluded from sdist packagen
```

Comment:
Usually preferred to just use `fetchFromGitHub` then.

Reply:
I thought that was less preferable due to the automated update bot will only look at pypi metadata.

If you want me to change, i can.

Reply:
nixpkgs-update is equipped to update this

Reply:
Okay, I will change then :)

Reply:
Looks like he runs all his tests through another package of his called `invocations`, problem is that they are circular dependencies of each other, and the dependency management  on invocations is.... [loose](https://github.com/pyinvoke/invocations/blob/bbf1b319bd1536817d5301ceb9eeb2f31830e5dc/setup.py#L11). I'll just open a PR on his repo to fix the setup.py and it should be easier to package and test.